README.adoc

```diff
@@ -52,6 +52,35 @@ and configure Maven to run it after building the application.
 [role='command']
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+To try out the application, first navigate to the `finish` directory and then run the following Maven goals to build the application and run it inside Open Liberty:
+[role='command']
+```
+mvn install
+```
+
+This command builds the application and creates a `.war` file in the target directory. It also
```

Comment:
I don't think this paragraph is needed for "try what you'll build".

Reply:
I think we agreed to keep this paragraph in because this is an intro guide about maven so we wanted to go more in depth into how mvn install works.